en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
- 
-![GPS](../../images/flight_modes/GPS_s.png) **RETURN: Return to Launch (RTL) / Return to Home**
+
+#### Return Mode (Return to Launch/Home - RTL)
+
+![GPS](../../images/flight_modes/GPS_s.png)
 
 The copter or multicopter will return in a straight line at the current altitude (if higher than the home position + hovering altitude) or on the hovering altitude (if higher than the current altitude), then lands automatically.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **MISSION**
 
-In Mission mode the aircraft will start the execution of a predefined Mission or Flight Plan. The Mission can be edited and uploaded to the aircraft with the software called Ground Control Station. The PX4 GCS is called QGroundControl and it can run on Windows, Linux, Android and iOS. You can download it from qgroundcontrol.com. The Mission upload to the aircraft can be done via USB connection on the ground or by wireless Data Link sometimes called Telemetry. During flight you can edit and change the current mission via this Data Link and monitor the current flight status and telemetry data live.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **TAKE OFF**
+#### Mission Mode
 
-When **Take Off** is activated the multicopter automatically will give thrust to the motors such to take off then climb vertically to a preset take off altitude, stop the climb and hover at place witing for pilot's command input.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **LAND**
+![GPS](../../images/flight_modes/GPS_s.png)
+
+In Mission mode the aircraft will start the execution of a predefined mission or flight plan. The mission can be edited and uploaded to the aircraft with a Ground Control Station (GCS).
+
+> **Tip** The PX4 GCS is called [QGroundControl](https://docs.qgroundcontrol.com/en/).
+
+#### Takeoff Mode
+
+![GPS](../../images/flight_modes/GPS_s.png)
+
+When **Take Off** is activated the copter automatically will give thrust to the motors such to take off then climb vertically to a preset take off altitude, stop the climb and hover at place waiting for pilot's command input.
+
+#### Land Mode
+
+![GPS](../../images/flight_modes/GPS_s.png)
 
-When **Land** is activated the multicopter automatically lower the throttle such that to stard slowly descending with preset vertical speed. When it touches the ground it will lower the throttle to minimum and disarm the craft.
+When **Land** is activated the copter automatically lower the throttle such that to start slowly descending with preset vertical speed. When it touches the ground it will lower the throttle to minimum and disarm the craft.
```

Comment:
Note that I have used copter to refer to any type of copter. Perhaps a cheat, but I think it reads better than multicopter, and everyone knows what it means. If that is not acceptable, then I'd be happy to refer to it as Multi-Copter (MC) in first instance, and as MC everywhere else.

Reply:
Agree!